src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.AgentPool.ps1

```diff
@@ -42,6 +44,14 @@ class AgentPool: ADOSVTBase
         if ([Helpers]::CheckMember($this.ControlSettings, "Agentpool.CheckForInheritedPermissions") -and $this.ControlSettings.Agentpool.CheckForInheritedPermissions) {
             $this.checkInheritedPermissionsPerAgentPool = $true
         }
+
+        if ($null -eq [AgentPool]::AllowedMemberCountInBroaderGroups)
+        {
+            if ([Helpers]::CheckMember($this.ControlSettings, "AgentPool.CheckForBroadGroupMemberCount") -and [Helpers]::CheckMember($this.ControlSettings, "AgentPool.AllowedBroadGroupMemberCount")) {
```

Comment:
check only for "CheckForBroadGroupMemberCount" and "AllowedBroadGroupMemberCount" 
eg.[Helpers]::CheckMember($this.ControlSettings.AgentPool, "CheckForBroadGroupMemberCount")